app/src/test/java/com/ucmobiledevelopment/freeways/IncidentTests.kt

```diff
@@ -1,43 +1,23 @@
 package com.ucmobiledevelopment.freeways
 
 import androidx.arch.core.executor.testing.InstantTaskExecutorRule
-import androidx.lifecycle.Observer
 import com.ucmobiledevelopment.freeways.dto.Incident
-import com.ucmobiledevelopment.freeways.dto.IncidentDTO
-import com.ucmobiledevelopment.freeways.dto.IncidentListDTO
 import com.ucmobiledevelopment.freeways.service.IncidentService
-import io.mockk.MockKAnnotations
-import io.mockk.coEvery
-import io.mockk.impl.annotations.MockK
 import junit.framework.Assert.*
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.newSingleThreadContext
-import kotlinx.coroutines.test.resetMain
 import kotlinx.coroutines.test.runTest
-import kotlinx.coroutines.test.setMain
-import org.junit.After
-import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.TestRule
-import java.util.*
-import java.util.concurrent.CountDownLatch
-import java.util.concurrent.TimeUnit
 import kotlin.collections.ArrayList
 
 class IncidentTests {
     @get:Rule
     var rule: TestRule = InstantTaskExecutorRule()
 
     lateinit var incidentService : IncidentService
-    //var allIncidents : IncidentListDTO? = IncidentListDTO(Results = ArrayList<List<IncidentDTO>>())
```

Comment:
Good suggestion.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.